frontend/admin/src/js/components/user/GeneralInformationTab/GeneralInformationTab.tsx

```diff
@@ -271,6 +272,49 @@ const AboutSection: React.FC<BasicSectionProps> = ({ user }) => {
         })}
       </span>
       <div data-h2-flex-item="b(2of3) l(1of3)">{user.telephone}</div>
+
+      <span data-h2-flex-item="b(1of3) l(1of6)">
+        {intl.formatMessage({
+          defaultMessage: "Member of CAF:",
+          description: "label for CAF status",
+        })}
+      </span>
+      <div data-h2-flex-item="b(2of3) l(1of3)">
+        {user.isVeteran === true &&
+          intl.formatMessage({
+            defaultMessage: "Is a veteran or member",
+            description: "user is a veteran or member",
+          })}
+        {user.isVeteran === false &&
+          intl.formatMessage({
+            defaultMessage: "Is not a veteran or member",
+            description: "user is not a veteran or member",
+          })}
```

Comment:
Not a huge deal but this could be simplified using a ternary:

```tsx
{user.isVeteran ? (
  intl.formatMessage({
    defaultMessage: "Is a veteran or member",
    description: "user is a veteran or member",
  })
) : (
  intl.formatMessage({
    defaultMessage: "Is not a veteran or member",
    description: "user is not a veteran or member",
  })
)}
```

Reply:
need to keep it the other way as that way it distinguishes false from null

Reply:
Ahhh, that makes sense. Thanks for clarifying.